vue/src/components/EpJulkaisuHistoriaJulkinen/EpJulkaisuLista.vue

```diff
@@ -29,6 +32,23 @@ export default class EpJulkaisuLista extends Vue {
 
   @Prop({ required: true })
   private latestJulkaisuRevision!: any;
+
+  get julkaisutFiltered() {
+    return _.chain(this.julkaisut)
+      .map(julkaisu => {
+        return {
+          ...julkaisu,
+          liitteet: this.filterLiitteet(julkaisu.liitteet),
```

Comment:
tämän voi myös tehdä
liitteet: _.filter(julkaisu.liitteet, liite => liite.kieli === this.$slang.value,

Reply:
Lisäksi kai kannattaa käyttää Kielet storea kälin kielien tutkimiseen 

Kielet.getSisaltoKieli.value